README.md

```diff
@@ -40,6 +40,15 @@ Arrays are allowed, but don't use them for sets of properties - only use them fo
 Test Set Sanitation
 -------------------
 
+### Setup
+
+(Requires `virtualenv`)
+```
+#> virtualenv -p python3 .env3
+#> . .env3/bin/activate
```

Comment:
Can you add the ``.env`` folder you used here to the ``.gitignore`` file, so that this is not checked in accidentally if someone creates the Python environment?

Reply:
Setup of what ?

Reply:
Setup for the test set sanitation, but that's already stated in the headline above, so there shouldn't be an additional need to expand the headline, should it?

Reply:
oh, I just noticed. 
So we have tests. we have a test tool. and we have a tool for that test tool to run and check the tests. ) 

would be nice to have a script then that will compile / get the latest version of testeth
and check all of it. but I think we should better discuss how to make such a tool which could be independent of aleth (which testeth depend). On devcon there will be a discussion about this.

Reply:
Would love to see this truely decoupled from aleth!